packages/render-fidelity-tools/src/components/rendering-scenario.ts

```diff
@@ -39,27 +41,66 @@ export class RenderingScenario extends LitElement {
     return `./results/${this.name}`;
   }
 
+  updated(changedProperties: Map<any, any>) {
+    super.updated(changedProperties);
+
+    if (this.analysis == null) {
+      this.loadAnalysis();
+    }
+  }
+
+  private toDecibels(rmsDistanceRatio: number) {
+    return (10 * Math.log10(rmsDistanceRatio)).toFixed(2)
```

Comment:
nit: I would've put the `toFixed` call inside the template literal rather than here.